utils/UninstallTemplate/Program.cs

```diff
@@ -212,16 +224,52 @@ private static string[] LaunchApp(string appName, string[] args)
                     CreateNoWindow = true,
                     UseShellExecute = false,
                     RedirectStandardOutput = true,
+                    RedirectStandardError = true
                 }
             };
 
+            process.Start();
+
             var lines = new List<string>();
+            process.OutputDataReceived += (sender, e) =>
+            {
+                if (e.Data == null)
+                {
+                    return;
+                }
+                lines.Add(e.Data);
+                Debug.WriteLine("output>>" + e.Data);
+                Console.WriteLine(e.Data);
+            };
+            process.BeginOutputReadLine();
 
-            process.Start();
-            while (!process.StandardOutput.EndOfStream)
+            var isError = false;
+            process.ErrorDataReceived += (sender, e) =>
+            {
+                if (e.Data == null)
+                {
+                    return;
+                }
+                isError = true;
+                Debug.WriteLine("error>>" + e.Data);
+                Console.WriteLine(e.Data);
+            };
+
+            process.BeginErrorReadLine();
+
+            process.WaitForExit(30000);
```

Comment:
Is there an option to reuse the `Consts.ExecutionTimeout` here?

Reply:
Yes. Done.